### ISSUE TITLE
letsencrypt support

### ISSUE THREAD

**Issue by OKNoah** [slypix/minio-dokku#3]
How would I get this to work with `dokku letsencrypt`. My hunch is it fails because it's on port 9000?

**Comment by slypix** [comment]
I haven't tried it with letsencrypt, however you should be able to configure it to use port 443 in the Dockerfile, by changing the `EXPOSE` to 443, and adding a custom port to minio server `--address :443`

you can try editing the two lines to read:
```
EXPOSE 443
CMD ["server", "--address :443", "/home/dokku/data"]
```

**Comment by mimischi** [comment]
@slypix I think this can be closed, if PR #4 is merged.

I just tested and running `dokku letsencrypt minio` works for me, after configuring the right ports, as mentioned in the PR.

To clarify, with Dokku there are two cases:

1. App has not been deployed yet:
One can immediately set the correct proxy port mapping for the app:
```bash
dokku proxy:ports-add minio http:80:9000
```

2. App has already been deployed

One needs to set the correct port mapping to the app and then remove the one that Dokku sets automatically (see PR #4).

```bash
dokku proxy:ports-add minio http:80:9000
dokku proxy:ports-remove minio http:80:5000
```

With this I got `dokku letsencrypt minio` working, running the `minio-dokku` installation via HTTPS.
